### ISSUE TITLE
React & Redux Group project: set up your kanban board

### ISSUE THREAD

**Issue by enoqueJonas** [enoqueJonas/space-travelers#25]
[Project](https://github.com/enoqueJonas/space-travelers/projects/1)

Note: I am solo for this week, how show I divide the tasks?

**Comment by Gambit142** [comment]
Hi @enoqueJonas 👋, 

Good job so far!

## Highlights
- Kanban board is a copy of the React Kanban Board ✔️ 
- Information about the number of students to work on this project was provided ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- Thank you for providing information about the number of students to work on this project. Since You are working alone, kindly do the following:
  - Provide information from Student Success or Microverse that you have no collaborators for all tasks to be undertaken this week.
  - Work on all tasks that have to do with `Missions` and not `rockets`. 💻 
  - Assign the `Mission` tasks to yourself. If you assign tasks to yourself, at the bottom right of each card representing a task, you will see your avatar. Currently, you have not assigned any tasks to yourself. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Gambit142** [comment]
Hi @enoqueJonas 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- The Kanban board is a copy of the React Kanban Board ✔️
- Information about the number of students to work on this project was provided ✔️
- Assigned all tasks for missions to yourself ✔️ 

#### There was an oversight, I didn't mention this in the first review. Kindly assign all group tasks to yourself before starting the project. 👍 😄 🙏  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
